recipes/gyptis/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "gyptis" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "f0a2ac4de0e18ee73de6930380caff3e35651faa1cc79baabebddedc910ea33d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/{{ name }}/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  skip: true  # [win]
+  {% set mpi_prefix = "mpi_" + mpi %}
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+  host:
+    - python
+    - pip
+    - fenics
+    - numpy
```

Comment:
Does this package builds against numpy? If so take a look at: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
No it doesn't but fenics does